packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -79,7 +79,7 @@ export default class QuanticFacet extends LightningElement {
    * @type  {'score' | 'alphanumeric' | 'occurrences' | 'automatic'}
    * @defaultValue `'automatic'`
```

Comment:
```suggestion
   * @defaultValue `'alphanumeric'`
```

Reply:
change default value for sortCriteria to 'alphanumeric'